### ISSUE TITLE
How to retrieve the selected values from a enumeration list

### ISSUE THREAD

**Issue by ppjmoors** [dixudx/rtcclient#137]
Hi

I'm looking for a way to find the selected values from an enumeration list. Is that possible with this client?

Example: I have a enumeration list various operating systems. In the editor I've defined a variable **SelOs** which is implemented as a enumeration picker-list. 
The question is: how can I determine the elements that are selected from the list of operating systems.

**Comment by dixudx** [comment]
@ppjmoors You can try to query the items, see [query examples](https://rtcclient.readthedocs.io/en/latest/advanced_usage.html#compose-your-query-string).

Also you can filter through [returned proporties](https://rtcclient.readthedocs.io/en/latest/quickstart.html#about-returned-properties) by yourself.

**Comment by ppjmoors** [comment]
I went through the complete list of examples but they didn't provide me additional information.

Let me try to explain again:
I have an enumeration _**AvailableOS**_ with operation systems values: MS-Windows, Fedora, Ubuntu, Solaris
In the editor I've implemented the _**SelectedOS**_ variable as Picker Enumeration List (same problem with Checkbox Enumeration List)
Next I started the browser and existing workitem
For that workitem (904) I selected for example value _Fedora_ in _**SelectedOS**_  and saved it. however when I add the _**SelectedOS**_ attribute to the return properties I get the content of _**AvailableOS**_.
This is what I did:
wk = myclient.getWorkitem(904)
wk.SelectedOS
Result: ['MSS-Windows', 'Fedora', 'Ubuntu', 'Solaris']

Can you explain to me what I possibly do wrong ?

**Comment by dixudx** [comment]
@ppjmoors If you've enabled `setup_basic_logging()`, you may find some log messages like this, `Get response from xxxxx`.

Then you can manually paste the xxxxx url in your broswer to get the raw xml content. By analysis and search in the raw xml, we may find out more clues.
